MSTest order tests by name (#42814)
dotnet/docs: docs/core/testing/order-unit-tests.md

:::code language="csharp" source="snippets/order-unit-tests/csharp/MSTest.Project/ByAlphabeticalOrder.cs":::

> Starting with MSTest 3.6, a new runsettings option was introduced to allow you to run tests by test names both in Test Explorers, and in the command line. To enable this feature, add the following to your runsettings file:

Reviewer: ```suggestion
Starting with MSTest 3.6, a new runsettings option lets you run tests by test names both in Test Explorers and on the command line. To enable this feature, add the `OrderTestsByNameInClass` setting to your runsettings file:
```
